test/e2e/serial/tests/workload_unschedulable.go

```diff
@@ -194,7 +194,7 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
 			}
 		})
 
-		It("[test_id:47617][tier2][unsched] workload requests guaranteed pod resources available on one node but not on a single numa", func() {
+		It("[test_id:47617][tier2][unsched][failalign] workload requests guaranteed pod resources available on one node but not on a single numa", Label("unsched", "failalign"), func() {
```

Comment:
why do we want to add the labels in the test description too?

Reply:
we probably should not anymore (I don't see us ever going back to ginkgo v1), I'll stop adding these and just use ginkgov2 `Label` from now on.

Reply:
We need to evaluate this indeed